service/create/service.go

```diff
@@ -28,8 +28,6 @@ import (
 )
 
 const (
-	ClusterListAPIEndpoint  string = "/apis/cluster.giantswarm.io/v1/awses"
-	ClusterWatchAPIEndpoint string = "/apis/cluster.giantswarm.io/v1/watch/awses"
```

Comment:
Some boyscouting, these constants weren't removed when we switched the TPR creation to operatorkit.